### ISSUE TITLE
Question about GPU

### ISSUE THREAD

**Issue by laolongboy** [JiahuiYu/generative_inpainting#181]
I try to use GPU for training but it don't work when using default setting：
`NUM_GPUS: 1
GPU_ID: -1`
The training process  only used the CPU.

Then I follow #68 #63 #14 to change ng.train.Trainer to  MultiGPUTrainer, add `num_gpus=config.NUM_GPUS` and set
`NUM_GPUS: 2
GPU_ID: [6,7]`
But the training process  still only used the CPU.

Did I ignore something?
Actually, my objective is to check the value of GAN_loss when training, cause I am trying to produce a pytorch version but my loss of discriminator and generator is very small(e-03) from beginning to end and the result is very blurry

**Comment by JiahuiYu** [comment]
You can comment lines in code about get_gpus, and manually set is via os.env CUDA VISIBLE DEVICES.

**Comment by TerminatorSd** [comment]
Hi, JiahuiYu. Sorry to bother you, but I have tried the following three methods, the gpu just does not work:
1. set GPU_ID = 0 or NUM_GPUS = 1 and GPU_ID = -1
2. comment lines in code about get_gpus and add code at the begining of train.py:
```
os.environ['CUDA_DEVICE_ORDER'] = 'PCI_BUS_ID' 
os.environ['CUDA_VISIBLE_DEVICES'] = '0'
```
3. enter on the command line: 
`CUDA_VISIBLE_DEVICES=0 python train.py`
How should i do?

**Comment by JiahuiYu** [comment]
Hello, GPU does not work means that the code is not running on GPU? Maybe you can check if you have properly install tensorflow-gpu.

**Comment by TerminatorSd** [comment]
Yes, you r right. tensorflow-gpu 1.13.0 does not work with cudnn 9.0, and I have installed tensorflow-gpu 1.12.0 instead. Thx a lot.
